JEMS-CSE416/JEMS
# Spaces

**Updates**
- Implemented .... 
- Fixed ....

**Testing**
- [x] Checked that values updated in MongoDB Database and images are stored in DO Space

**How to review:**
```
git checkout main
git fetch
git checkout spaces
cd client
npm install
npm start
```

1. Go to browser & create a new map: 
<img width="1625" alt="image" src="https://github.com/JEMS-CSE416/JEMS/assets/53535722/53c81475-f077-4900-bd65-67e4bbef6f81">

2. Page should be navigated to edit screen with created map click on settings modal
<img width="1626" alt="image" src="https://github.com/JEMS-CSE416/JEMS/assets/53535722/620dec0e-1231-4a07-a1cf-20986b15b878">

3. Upload an image and the image preview should display
<img width="1627" alt="image" src="https://github.com/JEMS-CSE416/JEMS/assets/53535722/b0ee2db4-bf49-43fe-9335-be9e882fae68">

4. See a notification that your map has been successfully updated. You can also verify by clicking on the settings button again to open the modal that displays your uploaded image.
<img width="1625" alt="image" src="https://github.com/JEMS-CSE416/JEMS/assets/53535722/e060f65b-bbb9-4554-956d-13ac9d05b7b6">

5. Check that image URL in backend MongoDB database was updated and viewable
<img width="1436" alt="image" src="https://github.com/JEMS-CSE416/JEMS/assets/53535722/3334954a-ac9c-4c7b-b63b-9a17a0235ff5">
